test/gateway/staker_proxy/self_destruct.js

```diff
@@ -0,0 +1,49 @@
+// Copyright 2019 OpenST Ltd.
```

Comment:
I would propose to test the followings in addition:
  - That contract was actually `selfdestruct -ed`. (web3.eth.getCode(address) can be helpful I would assume).
  - That remaining `ether` was sent to the owner.

Reply:
We can add contract code assertion, to ensure that `selfdestruct` is called. Asserting ether will be more like testing solidity `selfdestruct` works fine or not. Also, I don't think this contract can hold ether. There is not payable function.

Reply:
I've thought, like this:
  we do: `selfdestruct(owner)` and we want to test that, and not that we do `selfdestruct()` or `selfdestruct(non-owner-address)`. What do you think?